desktop/apps/auction2/routes.js

```diff
@@ -11,3 +12,18 @@ export const index = async (req, res) => {
   const newAuction = new Auction(sale)
   res.render('index', { auction: newAuction, me: me })
 }
+
+export const redirectLive = async (req, res, next) => {
+  const { sale } = await metaphysics({ query: SaleQuery(req.params.id) })
+  if (sale && sale.is_live_open) {
+    const liveUrl = `${PREDICTION_URL}/${sale.id}/login`
+    const { me } = await metaphysics({ query: MeQuery(req.params.id), req: req })
+    if (me && me.bidders && me.bidders.length > 0 && me.bidders[0].qualified_for_bidding) {
+      res.redirect(liveUrl)
+    } else {
+      return next()
```

Comment:
These returns are probably not necessary (assuming they were generated from a coffeescript conversion).